package.json

```diff
@@ -8,21 +8,23 @@
   "scripts": {
     "build": "yarn run clean && yarn run prep && NODE_ENV=production webpack --progress --colors",
     "clean": "rm -rf build/",
-    "chimp-test": "chimp chimp.config.js",
-    "chimp-test:watch": "yarn run chimp-test --watch",
+    "chimp-test": "concurrently --kill-others --success first \"NODE_ENV=test yarn run dev\" \"chimp chimp.config.js\"",
```

Comment:
Changing our test commands so that we don't set `NODE_ENV=test` every time we build just for dev. Now we only set it when truly in test mode

Reply:
In addition, `yarn run chimp-test` will now boot up `parity`, the dev server, and anything else we need. So you don't have to worry about running multiple commands.

Reply:
this is super nice, even if i know @jellegerbrandy doesn't like concurrently, because i want to have all the different processes run in multiple terminal windows!

Reply:
yes, exactly. Nothing against using concurrently, but I'd also like a command that just calls chimp without starting other processes (which I can start by hand, i.e. the original chimp-test command)

Reply:
ah, also, it messes up completely the circleci ouptut doing this stuff concurrently. It is useful there to start the processes explicitly separately, to debug the problems more easily.

Reply:
I will revert that change before merging 👍